packages/react/src/contents/components/base/media/components/image/Image.tsx

```diff
@@ -47,21 +44,18 @@ const Image = ({
           style={ImageStyled}
         />
         <noscript>
-          {fallbackSrc && <img src={fallbackSrc} alt={alt} {...props} />}
+          {fallbackSrc && (
+            <picture>
+              <img src={fallbackSrc} alt={alt} {...props} />
+            </picture>
+          )}
         </noscript>
       </picture>
     );
   };
 
   if (showAsThumbnail) {
-    return (
-      <img
-        alt={alt}
-        style={ImageThumbnail}
-        src={fallbackSrc}
-        data-test={`${name}-thumbnail`}
-      />
-    );
+    return <Thumbnail image={data.image} breakpoint={breakpoint}></Thumbnail>;
```

Comment:
You can close the tag in the first one if there is no children
`<Thumbnail image={data.image} breakpoint={breakpoint} />`

Reply:
done!